### ISSUE TITLE
`\big/ \Big/ \bigg/ \Bigg/` don't work anymore in Firefox

### ISSUE THREAD

**Issue by tmke8** [ronkok/Temml#71]
I see this output:

![image](https://github.com/user-attachments/assets/3798ca85-3b9a-4e56-9c4c-7aee21bc815c)

But I think it *used* to work in Firefox (it never worked in Chrome though).

Firefox 131.0.3, Ubuntu 24.04

**Comment by ronkok** [comment]
I agree that it has never worked in Chrome. Row184 in the Temml [Wiki Test](https://temml.org/tests/wiki-tests) has always been a problem. Until recently, I was more concerned with rows 178 & 179, which also displayed Chromium errors. But that Chromium bug has been fixed and now is a good time to focus on row 184.

The source of the problem is that the MathML Core operator dictionary does not give any solidus-type character the *stretchy* attribute. I've spent the last 1/2 hour experimenting and I find that I can **manually** set a stretchy attribute on `\` & `/`, Unicode characters U+002F & U+005C. Then I can set a `minsize` attribute to mimic LaTeX `\big`, `\Big` etc.

Results are mixed. The characters stretch properly in WebKit (Safari). Firefox stretches the solidus but not the reverse solidus. Chromium stretches both, but then inserts too much horizontal space between characters.

That's probably the best I'll be able to do. I'll work up a Temml revision later this week.
